conformance/tests/admin-network-policy-core-ingress-udp-rules.go

```diff
@@ -207,12 +208,65 @@ var AdminNetworkPolicyIngressUDP = suite.ConformanceTest{
 			// ensure ingress from slytherin is PASSED to hufflepuff at port 5353; ingressRule at index3 should take effect
 			success := kubernetes.PokeServer(t, s.ClientSet, &s.KubeConfig, "network-policy-conformance-slytherin", "draco-malfoy-0", "udp",
 				serverPod.Status.PodIP, int32(5353), s.TimeoutConfig.RequestTimeout, true)
-			assert.Equal(t, true, success)
+			assert.True(t, success)
 			// draco-malfoy-1 is our client pod in slytherin namespace
 			// ensure ingress from slytherin is ALLOWED to hufflepuff for rest of the traffic; matches no rules hence allowed
 			success = kubernetes.PokeServer(t, s.ClientSet, &s.KubeConfig, "network-policy-conformance-slytherin", "draco-malfoy-1", "udp",
 				serverPod.Status.PodIP, int32(53), s.TimeoutConfig.RequestTimeout, true)
-			assert.Equal(t, true, success)
+			assert.True(t, success)
 		})
 	},
 }
+
+var AdminNetworkPolicyIngressNamedPort = suite.ConformanceTest{
+	ShortName:   "AdminNetworkPolicyIngressNamedPort",
+	Description: "Tests support for ingress traffic on a named port using admin network policy API based on a server and client model",
+	Features: []suite.SupportedFeature{
+		suite.SupportAdminNetworkPolicy,
```

Comment:
same here, we probably want to decide on whether namedPorts is core or extended feature?

Reply:
If you look at https://github.com/kubernetes-sigs/gateway-api/blob/main/conformance/utils/suite/features.go and some test file like https://github.com/kubernetes-sigs/gateway-api/blob/main/conformance/tests/gateway-with-attached-routes.go, it seems to me that for extended testcases we do put the core feature and the specific extended feature together in the features list